jd4/sandbox.py

```diff
@@ -62,16 +62,19 @@ def _handle_reset_child():
 def _handle_compile(compiler_file, compiler_args, output_file, cgroup_file):
     pid = fork()
     if not pid:
-        chdir('/out')
-        os_close(STDIN_FILENO)
-        if output_file:
-            fd = os_open(output_file, O_WRONLY)
-            dup2(fd, STDOUT_FILENO)
-            dup2(fd, STDERR_FILENO)
-            os_close(fd)
-        if cgroup_file:
-            enter_cgroup(cgroup_file)
-        execve(compiler_file, compiler_args, SPAWN_ENV)
+        try:
```

Comment:
@iceboy233 Is my added `try` necessary? What if this code throws an exception? Is this a bug?